### DISCUSSION TITLE
Feature Request / Integration Request: wakatime.com

### DISCUSSION THREAD

**Discussion by PeterKnealeBizCover** [gitextensions/gitextensions#9892] [Ideas]
https://wakatime.com/ provides great insights into where your time goes,. It would be great to have a gitextensions plugin for it.

![image](https://user-images.githubusercontent.com/46661740/157552150-74a8e1d4-de9b-4d57-a864-64196ae4b19b.png)

**Comment by RussKie** [comment]
Hi @PeterKnealeBizCover, long time no see :) Thanks for the suggestion. The Team no longer accepts or takes on more plugins that ship as part of the app bundle, and we've been unbundling some of the original plugins. The 3rd plugins (e.g., https://github.com/topics/gitextensions-plugin) can now be installed via the [Plugin Manager plugin](https://github.com/gitextensions/gitextensions.pluginmanager), which is shipped with the app. We also provide a [plugin template](https://github.com/gitextensions/gitextensions.plugintemplate) for anyone who wants to build their own plugin.
Hope this helps.
